packages/documentation/pages/usage/components/filter.vue

```diff
@@ -0,0 +1,310 @@
+<template>
+	<div>
+		<ComponentInfo v-bind="{ component }" />
+
+		<KtTranslationContext :locale="settings.locale">
+			<div class="overview">
+				<div class="overview__component">
+					<h4>Component</h4>
+					<KtFilter
+						v-model="filters"
+						:columns="componentProps.columns"
+						:isLoading="componentProps.isLoading"
+					/>
+					<div class="overview__component__value">
+						<strong>Value</strong>: <span v-text="JSON.stringify(filters)" />
+						<a @click.prevent="reset">reset</a>
+					</div>
+				</div>
+				<div class="overview__code">
+					<h4>Code</h4>
+					<pre v-text="componentCode" />
+				</div>
+			</div>
+			<KtForm v-model="settings">
+				<div class="wrapper">
+					<div>
+						<h4>Settings</h4>
+						<KtFieldSingleSelect
+							formKey="locale"
+							helpText="Can be set via KtTranslationContext"
+							hideClear
+							label="Language"
+							leftIcon="global"
+							:options="[
+								{ label: 'German (de-DE)', value: 'de-DE' },
+								{ label: 'English (en-US)', value: 'en-US' },
+								{ label: 'Spanish (es-ES)', value: 'es-ES' },
+								{ label: 'French (fr-FR)', value: 'fr-FR' },
+								{ label: 'Japanese (ja-JP)', value: 'ja-JP' },
+							]"
+							size="small"
+						/>
+						<KtFieldToggleGroup
+							formKey="booleanFlags"
+							isOptional
+							label="Boolean Flags"
+							:options="[{ key: 'isLoading', label: 'isLoading' }]"
+							size="small"
+							type="switch"
+						/>
+					</div>
+					<div>
+						<h4>Texts</h4>
+						<KtFieldText
+							formKey="searchPlaceholder"
+							isOptional
+							label="placeholder"
+							size="small"
+						/>
+					</div>
+				</div>
+			</KtForm>
+		</KtTranslationContext>
+	</div>
+</template>
+
+<script lang="ts">
+import { Kotti, KtFilter } from '@3yourmind/kotti-ui'
+import { computed, defineComponent, ref } from '@vue/composition-api'
+import cloneDeep from 'lodash/cloneDeep'
+
+import ComponentInfo from '~/components/ComponentInfo.vue'
+
+const generateCode = (props: object) =>
+	[
+		'<KtFilter',
+		...Object.entries(props)
+			.sort(([a], [b]) => a.localeCompare(b))
+			// eslint-disable-next-line @typescript-eslint/no-unused-vars
+			.filter(([_, value]) => value !== null && value !== false)
+			.filter(([key, value]) => !(key === 'size' && value === 'medium'))
```

Comment:
also here you can use `KottiField.Size.Medium`
I'm surprised this didn't show up when we rebased yesterday after the enumeration PR?